packages/ui-map-components/src/components/TilePicker/TileControl.tsx

```diff
@@ -18,8 +18,7 @@ const StyledButton = styled(Button)<{
   display: block;
   pointer-events: auto;
   box-shadow: none;
-  background: ${({ theme }) =>
-    theme.palette.type === 'light' ? 'rgba(255, 255, 255, 0.9)' : 'rgba(43, 45, 56, 0.9)'};
+  background: #34353f;
```

Comment:
If we set one of the mui theme variables to be this black then we should be able to refer to it here. The main reason being that themes can change